cleaner.go

```diff
@@ -145,11 +153,6 @@ func (c *Cleaner) RemoveResources() error {
 			Group:   "addons.kyma-project.io",
 			Version: "v1alpha1",
 		},
-		{
-			Kind:    "ClusterAddonsConfiguration",
```

Comment:
this is also cluster scoped CRD so we can save some network calls by not trying to delete for every namespace